components/mpas-ocean/src/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -200,6 +204,9 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, layerThickness, layerThickEdgeMea
       nCells = nCellsArray(2)
       nCellsP1 = nCellsArray(size(nCellsArray)) + 1
 
+      use_quasi_monotone = config_Redi_use_quasi_monotone_limiter
+      limiter_safety_factor = config_Redi_quasi_monotone_safety_factor
+
```

Comment:
could these few lines be moved to redi_init?

Reply:
Happy to do that, but on this is it better to use module-level variables, or to have them as locals on the individual subroutine's stack?

Reply:
I think there's a preference for module-level variables (and logical or integer variables over strings, where applicable).